test/common/README.md

```diff
@@ -116,6 +116,14 @@ Checks whether `IPv6` is supported on this platform.
 
 Checks if there are multiple localhosts available.
 
+### hijackStderr/Stdout(listener)
```

Comment:
I'd rather keep the style consistent throughout, so would really prefer them separate.